From cdc25e486f781ad3704fa85d142855eb3f2c0443 Mon Sep 17 00:00:00 2001
From: Hauke Mehrtens <hauke.mehrtens@intel.com>
Date: Fri, 13 Sep 2019 16:49:47 +0200
Subject: [PATCH] PONRTSYS-5007: Fix checkpatch errors and warnings

Most of the fixes are only in the documentation.

The enum type DP_F_DATA_CONTINOUS_Q_RESV was renamed to
DP_F_DATA_CONTINUOUS_Q_RESV to fix a typo which was found by checkpatch,
this constant is never used in the kernel code.
---
 include/net/datapath_api.h | 60 +++++++++++++++++++++++-----------------------
 1 file changed, 30 insertions(+), 30 deletions(-)

diff --git a/include/net/datapath_api.h b/include/net/datapath_api.h
index a955407af60a..9c38128b4f86 100644
--- a/include/net/datapath_api.h
+++ b/include/net/datapath_api.h
@@ -368,8 +368,9 @@ typedef struct dp_subif {
 	union {
 		u16 def_qid; /*!< [out] def physical queue id assigned by DP */
 		u16 def_qlist[DP_MAX_DEQ_PER_SUBIF]; /*!< [in/out] default
-						      * physical qid list 
-						      * assigned by DP */
+						      * physical qid list
+						      * assigned by DP
+						      */
 	};
 	int lookup_mode; /*!< [out] CQM lookup mode for this device
 			  *   (dp_port based)
@@ -788,7 +789,7 @@ struct dp_subif_cache {
 };
 
 struct dp_gpid_tx_info {
-	/* Note for GPID max packet lengh setting, DP should get it from
+	/* Note for GPID max packet length setting, DP should get it from
 	 *      struct dp_tx_ring's tx_pkt_size
 	 */
 	u32 f_min_pkt_len : 1; /*!< flag to indicate whether min_pkt_len is
@@ -896,9 +897,9 @@ enum dp_port_data_flag {
 					*   packet from PP QOS port via credit
 					*   left and credit add
 					*/
-	DP_F_DATA_CONTINOUS_Q_RESV = BIT(7) /*!< reserve continous physical
-					     *   queue ID
-					     */
+	DP_F_DATA_CONTINUOUS_Q_RESV = BIT(7) /*!< reserve continuous physical
+					      *   queue ID
+					      */
 };
 
 /*! @brief typedef struct dp_port_data */
@@ -940,7 +941,7 @@ struct dp_port_data {
  *  |     |    (via UMT RX Msg to Alloc )       |           |
  *  |-----| ----------------------------------> |-----------|
  *
- *  4 Ring with Multipe QOS dequeue ports
+ *  4 Ring with Multiple QOS dequeue ports
  *  |-----| ------Multiple TXIN (Pkt+Desc)----> |-----------|
  *  |     |    (Poll Each QOS port Credit_Add ) |           |
  *  |     |                                     |           |
@@ -1036,8 +1037,6 @@ enum DP_DEV_DATA_FLAG {
  *\note
  */
 
-
-
 enum DP_RXOUT_QOS_MODE {
 	DP_RXOUT_BYPASS_QOS_ONLY = 0, /*!< bypass QOS but with FSQM */
 	DP_RXOUT_QOS, /*!< with QOS */
@@ -1055,7 +1054,7 @@ struct dp_rx_ring {
 				 *   for Peripheral device to enqueue packets to
 				 *   Host side via CQM or DMA in burst.
 				 *   If 0, then auto set by DP/CQM.
-				 *   Note: this paramater will be forced to tune
+				 *   Note: this parameter will be forced to tune
 				 *         down to HW capability
 				 */
 	u32   out_enq_port_id; /*!< [out] CQM enqueue port based ID */
@@ -1226,25 +1225,26 @@ struct dp_tx_ring {
 struct dp_dev_data {
 	enum DP_DEV_DATA_FLAG flag_ops; /*!< flag operation, for subif
 					 * registration
-					 * refer to enum DP_DEV_DATA_FLAG 
+					 * refer to enum DP_DEV_DATA_FLAG
 					 */
 	u8 num_rx_ring;   /*!< [in] number of rx ring from DC device to Host.
-			    *   num_rx_ring requirement:
-			    *   @num_rings <= @DP_RX_RING_NUM
-			    *   GRX350/PRX300:1 rx ring
-			    *   LGM: up to 2 rx ring, like Docsis can use 2 rings
-			    *   For two ring case:
-			    *    1st rxout ring without qos
-			    *    2nd rxout ring with qos
-			    */
+			   *   num_rx_ring requirement:
+			   *   @num_rings <= @DP_RX_RING_NUM
+			   *   GRX350/PRX300:1 rx ring
+			   *   LGM: up to 2 rx ring, like Docsis can use 2 rings
+			   *   For two ring case:
+			   *    1st rxout ring without qos
+			   *    2nd rxout ring with qos
+			   */
 	u8 num_tx_ring;   /*!< [in] number of tx ring from Host to DC device
-			    *   num_rx_ring requirement:
-			    *   @num_rings <= @DP_TX_RING_NUM
-			    *   Normally it is 1 TX ring only.
-			    *   But for 5G, it can support up to 8 TX ring
-			    *   For docsis, alhtough it is 16 dequeue port to WIB.
-			    *   But the final ring only 1, ie, WIB to Dcosis
-			    */
+			   *   num_rx_ring requirement:
+			   *   @num_rings <= @DP_TX_RING_NUM
+			   *   Normally it is 1 TX ring only.
+			   *   But for 5G, it can support up to 8 TX ring
+			   *   For docsis, alhtough it is 16 dequeue port to
+			   *   WIB.
+			   *   But the final ring only 1, ie, WIB to Dcosis
+			   */
 	u8 num_umt_port;   /*!< [in] number of UMT port.
 			    *    Normally is 1 only. But Docsis can use up to 2
 			    */
@@ -1279,7 +1279,7 @@ struct dp_dev_data {
 			 * dp_register_dev
 			 */
 	u16 max_gpid;  /*!< [in] maximum subif required which will be mapped to
-			* PP continious GPID block. The continuous limitation is
+			* PP continuous GPID block. The continuous limitation is
 			* from GSWIP (subif->GPID mapping design), not because
 			* of PP itself.
 			* since overall number of GPID < nubmer of CTP in HW,
@@ -1833,9 +1833,9 @@ static inline struct umt_ops *dp_get_umt_ops(int inst)
 }
 
 enum DP_DATA_PORT_TYPE {
-        DP_DATA_PORT_LINUX = 0, /*!< data port owned by Linux */
-        DP_DATA_PORT_DPDK, /*!< data port owned by DKDP */
-        DP_DATA_PORT_MPE, /*!< data port owned by MPE */
+	DP_DATA_PORT_LINUX = 0, /*!< data port owned by Linux */
+	DP_DATA_PORT_DPDK, /*!< data port owned by DKDP */
+	DP_DATA_PORT_MPE, /*!< data port owned by MPE */
 };
 
 /*!
